Strings/Cards+Octopus.jsonc

```diff
@@ -2,34 +2,34 @@
 
 	"オクトパスカードのオフセットを変更": {
 		"ja": "オクトパスカードのオフセットを変更",
-		"en": "Change the Offset of this Octopus Card",
-		"zh-Hans": "更改此八达通的抵销金额",
-		"zh-Hant": "更改此八達通的抵銷金額",
-		"zh-HK": "更改此八達通的抵銷金額"
+		"en": "Change the convenience limit of this Octopus",
+		"zh-Hans": "更改此八达通的备用限额",
+		"zh-Hant": "更改此八達通的備用限額",
+		"zh-HK": "更改此八達通的備用限額"
 	},
 
 	"オフセットを設定": {
 		"ja": "オフセットを設定",
-		"en": "Set the Card Offset",
-		"zh-Hans": "设定抵销金额",
-		"zh-Hant": "設定抵銷金額",
-		"zh-HK": "設定抵銷金額"
+		"en": "Set the convenience limit",
+		"zh-Hans": "设定备用限额",
+		"zh-Hant": "設定備用限額",
+		"zh-HK": "設定備用限額"
 	},
 
 	"このオクトパスカードにオフセットは設定されていません。": {
 		"ja": "このオクトパスカードにオフセットは設定されていません。",
-		"en": "This Octopus Card has no offset.",
-		"zh-Hans": "此八达通没有抵销金额。",
-		"zh-Hant": "此八達通沒有抵銷金額。",
-		"zh-HK": "此八達通沒有抵銷金額。"
+		"en": "The convenience limit of this Octopus has not been set.",
```

Comment:
It's a little weird to see 'Octopus' used instead of 'Octopus Card'

Reply:
It's referred to officially as simply "Octopus", that's why I made the change. [https://www.octopus.com.hk/en/consumer/octopus-cards/products/on-loan/standard.html](url)

Reply:
There are places where it's referred to as both Octopus Card and Octopus, so it's a little confusing.
https://www.octopus.com.hk/en/consumer/octopus-cards/about/index.html

Reply:
Well, their website is a little inconsistent, but in Chinese the "card" (卡) is normally omitted. In the official [Conditions of Issue of Octopus](https://www.octopus.com.hk/en/document/conditions_of_issue.pdf), it's also referred to as "Octopus," including different types of Octopus such as "On-Loan Octopus", "Sold Octopus", etc.

I'm fine with adding it back though, even [Apple](https://www.apple.com/hk/en/apple-pay/octopus/) can't decide to call it "Octopus" or "Octopus card". I just thought it'd be a bit more consistent with the Chinese translations.

Reply:
Ah, I understand. I only thought to keep it because we use 'card' in other places so it would be more consistent, but I think 'card' is already implied just by using the app, so it's probably alright.
<img src="https://user-images.githubusercontent.com/6291467/95685531-2d2a7a00-0c44-11eb-81c4-4e8e742f9070.png" width="300"/>

Thank you for clarifying.